persistence-cassandra/src/main/scala/com/evolutiongaming/kafka/flow/cassandra/CassandraPersistence.scala

```diff
@@ -26,10 +27,11 @@ object CassandraPersistence {
     session: scassandra.CassandraSession[F],
     sync: CassandraSync[F],
     consistencyOverrides: ConsistencyOverrides,
-    keysSegments: KeySegments
+    keysSegments: KeySegments,
+    tablesTtl: Option[FiniteDuration] = None
```

Comment:
let's name it `journalsTtl` please so that it's clearer

Reply:
The idea was to use same ttl for all three tables

Reply:
I don't think that would be flexible enough to be honest, different tables probably will have different requirements in regards to TTLs